### PR TITLE
time machine clusters

### PR DESCRIPTION
**Pull request by Abhishekism9450**

### CODE REVIEW

File: nutanix/resource_nutanix_ndb_time_machine_cluster.go [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+	conn := meta.(*Client).Era
+
+	tmsID := d.Get("time_machine_id")
+	clsID := d.Get("nx_cluster_id")
+	resp, err := conn.Service.ReadTimeMachineCluster(ctx, tmsID.(string), clsID.(string))
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	if err = d.Set("log_drive_id", resp.LogDrive); err != nil {
+		return diag.Errorf("error setting log_drive_id for tms clusters %s: %s", d.Id(), err)
```
Comment (siddharth-kulshrestha) [inline nutanix/resource_nutanix_ndb_time_machine_cluster.go]: got following error while setting  <attr> for time machine cluster with id: %s

OR 

error occurred while setting  <attr> for time machine cluster with id: %s

Comment (siddharth-kulshrestha) [reply nutanix/resource_nutanix_ndb_time_machine_cluster.go]: Make changes everywhere.

File: nutanix/resource_nutanix_ndb_time_machine_cluster.go [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+	if d.HasChange("sla_id") {
+		updateReq.SLAID = utils.StringPtr(d.Get("sla_id").(string))
+		updateReq.ResetSLAID = utils.BoolPtr(true)
+	}
+
+	if d.HasChange("nx_cluster_id") {
+		updateReq.NxClusterID = utils.StringPtr(d.Get("nx_cluster_id").(string))
+	}
+
+	// update Call
```
Comment (siddharth-kulshrestha) [inline nutanix/resource_nutanix_ndb_time_machine_cluster.go]: Give more elaborative comments.

File: nutanix/resource_nutanix_ndb_time_machine_cluster.go [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+
+	_, er := conn.Service.UpdateTimeMachineCluster(ctx, tmsID.(string), clsID.(string), updateReq)
+	if er != nil {
+		return diag.FromErr(er)
+	}
+	log.Printf("NDB Time Machine Cluster with %s id is updated successfully", d.Id())
+	return resourceNutanixNDBTmsClusterRead(ctx, d, meta)
+}
+func resourceNutanixNDBTmsClusterDelete(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
```
Comment (siddharth-kulshrestha) [inline nutanix/resource_nutanix_ndb_time_machine_cluster.go:213]: Give a line space in between. 
Is lint not complaining about this?

**Review by siddharth-kulshrestha — CHANGES_REQUESTED** [review]

**Review by siddharth-kulshrestha — APPROVED** [review]

File: nutanix/resource_nutanix_ndb_time_machine_cluster.go (hunk 1 above)
Comment (Abhishekism9450) [reply nutanix/resource_nutanix_ndb_time_machine_cluster.go]: sure.

File: nutanix/resource_nutanix_ndb_time_machine_cluster.go (hunk 3 above)
Comment (Abhishekism9450) [reply nutanix/resource_nutanix_ndb_time_machine_cluster.go:213]: sure. 
No, Lint didn't complain about the spacing.

**Review by abhimutant — APPROVED** [review]

**Pull request merged**